test/unittests/VisitTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * Unit test for Visit class
+ *
+ * PHP Version 5
+ *
+ * @category Tests
+ * @package  Test
+ * @author   Mélanie Legault <melanie.legault2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+require_once __DIR__ . '/../../vendor/autoload.php';
+require_once __DIR__ . '/../../php/libraries/VisitController.class.inc';
+require_once __DIR__ . '/../../php/libraries/Visit.class.inc';
+
+use \LORIS\Visit;
+use \LORIS\VisitController;
+use \PHPUnit\Framework\TestCase;
+
+/**
+ * Unit test for Candidate class
+ *
+ * @category Tests
+ * @package  Test
+ * @author   Mélanie Legault <melanie.legault2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class VisitTest extends TestCase
+{
+    /**
+     * Visit object use in tests
+     *
+     * @var Visit
+     */
+    private $_visit;
+
+    protected function setUp()
+    {
+/*        parent::setUp();
```

Comment:
Commented code still needed?

Reply:
not really, different version of try. will clean up when every thing is fine with unit test